.github/ISSUE_TEMPLATE/jetbrains_feedback.md

```diff
@@ -0,0 +1,12 @@
+---
+name: '❌ Do not submit feedback or feature requests here'
```

Comment:
Could we be friendlier here? Someone is trying to give us the gift of feedback or a feature request... can we be less ❌ and *not* and more positive, motivating "to get your feedback handled by the right team" or something like that.

Where does the "about" field get rendered? Can it render a link? Or could we put (link in description) or something like that?

Reply:
ok!!! thanks for the feedback, I made it more friendly :) 

I kept the ❌ as it's a **bold** eye catcher, but the wording should sound more friendly now!